optimum/bettertransformer/transformation.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2022 The HuggingFace and Meta Team.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from copy import deepcopy
+from typing import Optional
+
+import torch
+
+from optimum.utils import is_accelerate_available
+
+from .models import is_module_fast
+from .utils import check_if_pytorch_greater_112
```

Comment:
Maybe put this in `optimum.utils` to make it available for every subpackage.

Reply:
Fixed in 77d0a35889f0356f3bb8605ac9d098a35aef0697